composer/algorithms/no_op_model/__init__.py

```diff
@@ -1,3 +1,8 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""Replaces model with a dummy model of type :class:`NoOpModelClass` on Event.INIT
```

Comment:
Can you elaborate how applying this algorithm can be useful for dataloader profiling?